const ApiError = require("../exceptions/api-error");

class ChatController {
    async getMessages(req, res, next) {
        try {

        } catch (e) {
            next(e);
        }
    }
    async sendMessage(req, res, next) {
        try {

        } catch (e) {
            next(e);
        }
    }
}

module.exports = new ChatController();